### PR TITLE
Yixuan CSS Battle

### PR DESCRIPTION
**Pull request by jyxhazcake**

### CODE REVIEW

File: src/CSSBattle/2_Carrom/Yixuan.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<div></div>
+<div></div>
+<div></div>
+<style>
+  body {
+    background: #62374e;
+    display: grid;
+    gap: 100px;
+    margin: 50px;
+    grid-template-columns: 150px auto;
```
Comment (kktai1512) [inline src/CSSBattle/2_Carrom/Yixuan.html:11]: nice use of grid

File: src/CSSBattle/8_ForkingCrazy/Yixuan.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<div class="handle"></div>
+<div class="base"></div>
+<div class="prong one"></div>
```
Comment (kktai1512) [inline src/CSSBattle/8_ForkingCrazy/Yixuan.html]: for the prongs perhaps can use box shadow trick to repeat prongs

**Review by limwenfeng — APPROVED** [review]
LGTM (Review 1)

File: src/CSSBattle/8_ForkingCrazy/Yixuan.html (hunk 2 above)
Comment (jyxhazcake) [reply src/CSSBattle/8_ForkingCrazy/Yixuan.html]: Updated in commit f943ede, nice stuff

**Pull request merged**